CONTRIBUTING.md

```diff
@@ -68,11 +68,6 @@ You may be asked to clarify things or try different approaches, so please follow
 We'd love to accept your contributions to iTwin.js.
 There are just a few guidelines you need to follow.
 
-### Contributor License Agreement (CLA)
```

Comment:
Why remove this CLA piece?

Reply:
It's still valid for the entire Org, not sure why the mobile-idk-android is specifically called out in the setup. The "Linked Organizations" section covers all public repos.

Reply:
reverted